src/ui/Footer/Contact.jsx

```diff
@@ -2,22 +2,49 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
+import { isInternalURL } from '@plone/volto/helpers/Url/Url';
+
 const Contact = ({ children, contacts }) =>
   children?.length ? (
     children
   ) : (
     <div className="contact-block">
       {contacts?.map((contact, index) => (
         <div className="contact" key={index}>
-          <Link to={contact.link} className="bold">
-            {contact.text}
-          </Link>
+          {isInternalURL(contact.link) ? (
+            <Link to={contact.link} className="bold">
+              {contact.text}
+            </Link>
+          ) : (
+            <a
+              href={contact.link}
+              title={contact?.text}
+              target={'_blank'}
+              rel="noopener noreferrer"
+              className={'bold'}
+            >
+              {contact.text}
+            </a>
+          )}
           {contact.children && (
             <div className="subcontact">
               {contact.children.map((child, index) => (
-                <Link to={child.link} key={index}>
-                  {child.text}
-                </Link>
+                <>
+                  {isInternalURL(child.link) ? (
+                    <Link to={child.link} key={index}>
+                      {child.text}
+                    </Link>
+                  ) : (
+                    <a
+                      href={child.link}
+                      title={child?.text}
```

Comment:
@andreiggr  there is no need to have title on the link

Reply:
roger. Removed link in next commit